gpustat/test_gpustat.py

```diff
@@ -161,7 +163,11 @@ def _MockedProcess(pid):
 
 MOCK_EXPECTED_OUTPUT_FULL = """\
 [0] GeForce GTX TITAN 0 | 80'C,  16 %,  76 %,  125 / 250 W |  8000 / 12287 MB | user1:python/48448(4000M) user2:python/153223(4000M)
+ ├─ 48448 ( 85%,  3.14%): python
+ └─ 153223 ( 15%,  0.00%): python
```

Comment:
I would like pid numbers aligned at 6 characters for better layout.

Reply:
Fixed.